packages/mobile/src/escrow/saga.ts

```diff
@@ -37,56 +42,73 @@ import { getConnectedAccount, getConnectedUnlockedAccount } from 'src/web3/saga'
 const TAG = 'escrow/saga'
 
 function* transferStableTokenToEscrow(action: TransferPaymentAction) {
+  Logger.debug(TAG + '@transferToEscrow', 'Begin transfer to escrow')
   try {
-    const { phoneHash, amount, txId, tempWalletAddress } = action
-    const escrow = yield call(getEscrowContract, web3)
-    const stableToken = yield call(getStableTokenContract, web3)
-
-    const account = yield call(getConnectedUnlockedAccount)
+    const { phoneHash, amount, tempWalletAddress } = action
+    const escrow: Escrow = yield call(getEscrowContract, web3)
+    const stableToken: StableToken = yield call(getStableTokenContract, web3)
+    const account: string = yield call(getConnectedUnlockedAccount)
 
     Logger.debug(TAG + '@transferToEscrow', 'Approving escrow transfer')
     const convertedAmount = web3.utils.toWei(amount.toString())
     const approvalTx = stableToken.methods.approve(escrow.options.address, convertedAmount)
 
-    yield call(sendTransaction, approvalTx, account, TAG, txId)
+    yield call(sendTransaction, approvalTx, account, TAG, 'approval')
```

Comment:
Is the last argument just a comment? or does it need to have that specific value for this use case?

Reply:
The last value can be anything, it's only use is in the logs